src/cml.js

```diff
@@ -220,13 +220,7 @@ class CML {
   }
 
   async repo_token_check() {
-    try {
-      await this.runner_token();
-    } catch (err) {
-      throw new Error(
-        'REPO_TOKEN does not have enough permissions to access workflow API'
-      );
-    }
+    await this.runner_token();
```

Comment:
- old (wrong): `try { ABC(); } catch { throw "B errored" }`
- current (suboptimal): `ABC();`
- ideal: `try { ABC(); } catch (err) { if (err.msg == "unhelpful msg related to B") throw "B errored (original message: {err})"; else throw err }`

vis e.g. https://github.com/iterative/cml/pull/554#issuecomment-848161657